packages/annotations/src/annotations.ts

```diff
@@ -62,28 +62,52 @@ export function getAnnotationServiceUrl(
 export function searchAnnotations(
   requestOptions: IQueryFeaturesRequestOptions
```

Comment:
I think we should make querying the users (which is very expensive) is optional. I'm not sure if the best thing is to extend `IQueryFeaturesRequestOptions` w/ something like `include?: 'users'` (and then strip that before passing along) or take in a second argument.